ariadne/asgi.py

```diff
@@ -167,6 +168,15 @@ async def render_playground(  # pylint: disable=unused-argument
     ) -> Response:
         return HTMLResponse(PLAYGROUND_HTML)
 
+    def build_json_response(
```

Comment:
Please move this under `extract_data_from_multipart_request` and make it take `request: Request` as first argument.